src/web/app/components/question-types/question-edit-answer-form/num-scale-question-edit-answer-form.component.html

```diff
@@ -6,4 +6,8 @@
   <div>
     Possible Acceptable Values are: {{ possibleValues }}
   </div>
+  <div>
+    <p *ngIf="!isValidPossibleValue(responseDetails.answer) && responseDetails.answer != 0" style="margin-left: 1em;" class="text-danger">The scale entered is not a possible value.</p>
+    <p *ngIf="responseDetails.answer == 0" style="margin-left: 1em; color: blue;">Please enter a scale number.</p>
```

Comment:
For answer that is not submitted, shouldn't it be a special value `NUMERICAL_SCALE_ANSWER_NOT_SUBMITTED` (You can find the value by searching in the codebase)?

Reply:
The value stored for `NUMERICAL_SCALE_ANSWER_NOT_SUBMITTED` is -999, but it seems that the initial default value for answer not submitted is 0.